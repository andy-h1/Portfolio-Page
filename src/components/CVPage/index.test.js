import React from 'react';
import { render } from '@testing-library/react';
import { CVPage } from '.';

describe('CV Page Component', () => {
  it('should render', () => {
    const { container } = render(<CVPage />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
