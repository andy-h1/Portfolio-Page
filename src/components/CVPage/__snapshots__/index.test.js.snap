// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CV Page Component should render 1`] = `
<div
  class="sc-eCApGN fMFnsy"
>
  <a
    download="Andy Huynh CV"
    href="Andy-Huynh-CV.pdf"
  >
    <button
      class="sc-gKAblj hmyHXn"
      type="button"
    >
      Download CV
    </button>
  </a>
  <ul
    class="sc-jSFkmK ixcXzT"
  >
    <li
      class="sc-bdnylx lpqsMp"
    >
      <a
        href="https://deliveroo.co.uk/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="deliveroo"
          height="auto"
          src="deliveroo.png"
          width="60px"
        />
      </a>
      <h3
        class="sc-gtssRu gORUoG"
      >
        Tech Recruiter
      </h3>
      <h3
        class="sc-dlnjPT bUgsmE"
      >
        Nov 2018 - Present
      </h3>
      <p
        class="sc-hKFyIo fKwaFs"
      >
        I joined the Deliveroo recruitment team focused on hiring Product Managers, Product Marketing Managers and Software Engineers. When I joined we were around 20+ in the team and now have grown to over 60+ internationally.
      </p>
    </li>
    <li
      class="sc-bdnylx lpqsMp"
    >
      <a
        href="https://www.clearscore.com/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="clearscore"
          height="25px"
          src="clearscore.png"
          width="auto"
        />
      </a>
      <h3
        class="sc-gtssRu gORUoG"
      >
        Talent Acquisition
      </h3>
      <h3
        class="sc-dlnjPT bUgsmE"
      >
        Jan 2018 - Oct 2018
      </h3>
      <p
        class="sc-hKFyIo fKwaFs"
      >
        When I joined ClearScore, I was 1 out of 3 recruiters in the team. We grew the team from 100 employees to over 200 internationally when I left.
      </p>
    </li>
    <li
      class="sc-bdnylx lpqsMp"
    >
      <a
        href="https://arrowsgroup.com/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="arrowsgroup"
          height="90px"
          src="arrowsgroup.png"
          width="auto"
        />
      </a>
      <h3
        class="sc-gtssRu gORUoG"
      >
        Tech Recruiter
      </h3>
      <h3
        class="sc-dlnjPT bUgsmE"
      >
        April 2016 - Jan 2018
      </h3>
      <p
        class="sc-hKFyIo fKwaFs"
      >
        After working 4 years as a neonatal nurse, I wanted to pursue my interest in the world of tech and started this journey working as a Tech recruiter for Arrows Group.
      </p>
    </li>
    <li
      class="sc-bdnylx lpqsMp"
    >
      <a
        href="https://www.nhs.uk/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="nhs"
          height="40px"
          src="nhs.png"
          width="auto"
        />
      </a>
      <h3
        class="sc-gtssRu gORUoG"
      >
        Neonatal Nurse
      </h3>
      <h3
        class="sc-dlnjPT bUgsmE"
      >
        Jun 2012 - March 2016
      </h3>
      <p
        class="sc-hKFyIo fKwaFs"
      >
        After graduating from King's College London with a Bachelor in Paediatric Nursing, I embarked in a career as a neonatal nurse working for well-known hospitals across London.
      </p>
    </li>
  </ul>
</div>
`;
